includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -146,6 +146,7 @@ public function capture_terminal_payment( $request ) {
 					[ 'status' => $http_code ]
 				);
 			}
+			$order->add_meta_data( 'receipt_url', get_rest_url( null, 'wc/v3/payments/readers/receipts/' . $intent->get_id() ) );
```

Comment:
This new metadata is not used by the client directly and it would be good to provide a brief explanation in the comments on why it has to be stored.

Reply:
Good point, added in 68d2ee267031f12c61dff1fbe7d370170d280def.